backend/codesirius/problems/tests/test_tag_model.py

```
@@ -0,0 +1,19 @@
+from rest_framework.response import Response
+from rest_framework import status
+
+class CodesiriusAPIResponse(Response):
+    def __init__(self, data, status_code, message):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)